virtual-desktop/src/app/plugin-manager/plugin-factory/angular2/angular2-plugin-factory.ts

```diff
@@ -85,7 +85,8 @@ class SimpleAngularComponentFactory extends ComponentFactory {
 export class Angular2PluginFactory extends PluginFactory {
   private readonly logger: ZLUX.ComponentLogger = BaseLogger;
   private static getAngularModuleURL(pluginDefinition: MVDHosting.DesktopPluginDefinition): string {
-    return ZoweZLUX.uriBroker.pluginResourceUri(pluginDefinition.getBasePlugin(), 'main.js');
+    let entryPoint = pluginDefinition.getBasePlugin().getWebEntryPoint() || 'main.js';
```

Comment:
Is this intended behaviour? When webContent is not present, you set entryPoint to undefined (https://github.com/zowe/zlux-platform/pull/105/files#diff-46e106c34def1f9496bcb38ec7bd10d61c9631a6c8ee643a0ce96fbe6b46a8e0R187) , but this line 88 defaults to 'main.js' regardless then

Reply:
if webContent is not present, they have an invalid angular plugin and schema validation will fail it upon startup